src/tests/dbus-tests/test_70_encrypted.py

```diff
@@ -609,6 +609,53 @@ def test_convert_xfail(self):
 
         self.assertEqual(2, _get_luks_version(disk))
 
+    def test_header_backup_restore(self):
+        disk = self.vdevs[0]
+        device = self.get_device(disk)
+        self._create_luks(device, self.PASSPHRASE)
+
+        self.addCleanup(self._remove_luks, device)
+        self.udev_settle()
+
+        # check that backup normally works
+        BACKUP_FILE = "/var/tmp/udisks_encrypted_header_backup.luks"
```

Comment:
Please use [tempfile](https://docs.python.org/3.8/library/tempfile.html#tempfile.TemporaryDirectory) to create a temporary directory for this, it will take care of cleanup.